### PR TITLE
Add a code of conduct

### PR DESCRIPTION
**Pull request by SuperQ**
This is a technical code of conduct for maintinaing Noisebridge
infrastructure.
* Fix up some minor markdown formatting.

Signed-off-by: Ben Kochie <superq@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,35 @@
...
+
+Noisebridge infrastructure members follow the Rack Code of Conduct
+
+### Security
+
+Noisebridge infrastructure members protect the privacy and security of our systems and members.
+
+* Secrets should be checked in using ansible-vault.
+* The vault password should be kept safe.
+* Care needs to be taken when handling Issues and PRs to protect the privacy of members.
```
Comment (elimisteve) [inline README.md:15]: Like by not mentioning their real name in an issue?

**Comment by elimisteve** [comment]
Looks pretty great to me! @sunjam Thoughts?

File: README.md (hunk 1 above)
Comment (SuperQ) [reply README.md:15]: Yes, names, email addresses in aliases, etc.

**Comment by sunjam** [comment]
I don't know the backstory on the code of conduct or if it relates to some previous event or consensus.  Reads as related to just the repo contents.  I see the original post in this pull is signed with a name and email, but does that relate to the code of conduct.  I also see a list of maintainers.md file which links to github accounts with names and pictures.

**Comment by jof** [comment]
LGTM!

**Review by jof — APPROVED** [review]
LGTM

**Pull request merged**